apps/api/src/services/file_export/document_export/pdf_export.py

```
@@ -19,48 +19,23 @@ async def upload_file_to_s3(output_filename):
     )
 
 
-class PdfExport:
+class PdfExport(BasePdfExport):
+    """API PDF export that adds optional S3 upload after Celery PDF generation."""
+
     def __init__(self, title: str, html_string: str = None):
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)